src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/Internal/Topics.cs

```diff
@@ -195,5 +237,20 @@ public static object WrapIfNeeded(object message)
         {
             return message is RouterEnvelope ? new MediatorRouterEnvelope(message) : message;
         }
+
+        public static string MakeKey(IActorRef actorRef)
+        {
+            return MakeKey(actorRef.Path);
+        }
+
+        public static string EncodeName(string name)
+        {
+            return name == null ? null : Uri.EscapeDataString(name);
```

Comment:
Scala version doesn't have this check
https://github.com/akka/akka/blob/master/akka-cluster-tools/src/main/scala/akka/cluster/pubsub/DistributedPubSubMediator.scala#L281